docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,25 @@ Release Notes for BIG-IP Controller for Cloud Foundry
 Added Functionality
 ```````````````````
 
+v1.1.0
+------
+
+Added Functionality
+```````````````````
+* L7 (HTTP) per route configuration through Route Services when controller is run as a Service Broker
+
+  * Virtual Server for L7 (HTTP) route can have its own Policies, Profiles and SSL Profiles
+  * Pool for L7 (HTTP) route can have its own load balancing mode and Health Monitors
+
+* Virtual Server targeting Virtual Server architecture
```

Comment:
Not sure this should be included, as it seems like more of an internal knowledge kind of thing. Maybe change to:
* Adopts a new, two-tier architecture. See the `BIG-IP Controller for Cloud Foundry documentation </containers/latest/cloudfoundry/#overview>`_ for more information.